configuration.rst

```diff
@@ -767,7 +767,9 @@ the right situation:
   but the overrides only apply to one environment.
 
 *Real* environment variables always win over env vars created by any of the
-``.env`` files.
+``.env`` files. This behavior depends on
+`variables_order <http://php.net/manual/en/ini.core.php#ini.variables-order>`_ to
+contain an ``E`` to expose the ``$_ENV`` superglobal.
```

Comment:
Friendly ping @stof @nicolas-grekas

Reply:
Yes, let's wait for their review because this change is really important, so we must be sure it's correct. Thanks!

Reply:
The Dotenv has a fallback reading from `$_SERVER` (for env variables not  starting with `HTTP_` to avoid security issues due to confusion with HTTP headers), but I'm not sure the CLI populates it. So it should support the case of not having `E` in the order as long as you don't rely on env variables starting with `HTTP_`